src/main/resources/changesets/changelog-consent-100.0.xml

```diff
@@ -0,0 +1,17 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+    <changeSet id="100.0" author="qhoque">
+        <addColumn tableName="email_entity">
+            <column name="sendgrid_response" type="text"/>
+            <column name="sendgrid_status" type="integer" defaultValue="null"/>
+            // todo: default value of date_sent for existing records
+            <column name="create_date" type="date" defaultValue="date_sent"??/>
+        </addColumn>
+        <sql>
+        <!-- Populate date_sent column on email_entity table
+                 with date_sent value for existing records -->
+            UPDATE email_entity e SET create_date = date_sent;
```

Comment:
Had another thought here ... as a general rule, whenever we're updating a value, we should set a where condition so that we're only updating columns that are null. That prevents accidental overwrites. And we should also check that we're not putting a null into the create_date field. From looking at the databases, all values are populated, so there's no practical risk, but it is a good habit to get into.